src/views/examples/Register.js

```diff
@@ -24,145 +25,83 @@ import {
       console.log("Error while registering");
     }
-    }else{
-      setIsdisabled(false);
-    }
-       
-  };
-  const confirmpassword = async (e) => {
-    var class_valid = "is-valid form-control";
-    var class_invalid = "is-invalid form-control";
-
-    setConfirmPassword(e.value);
-    if(e.value!==password){
-      passState('Passwords did not match!');
-      e.className=class_invalid;
-    }else{
-      passState(' ');
-      setIsdisabled(false);
-      e.className=class_valid;
-    }
-  };
- 
+
+    };
+
+
+
+
   const dispatch = useDispatch();
-  
-  return ( 
+
+  return (
     <>
       <Col lg="6" md="8">
         <Card className="bg-secondary shadow border-0">
           <CardBody className="px-lg-5 py-lg-5">
-           
+
             <Form role="form" >
               <FormGroup>
                 <InputGroup className="input-group-alternative mb-3">
                   <InputGroupAddon addonType="prepend">
                     <InputGroupText>
-                      <i className="ni ni-email-83" />
-                    </InputGroupText>
-                  </InputGroupAddon>
-                  <Input required valid
-                    placeholder="IITK Email ID"
-                    type="email"
-                    autoComplete="off"
-                    onChange={(e) => validateEmail(e.target)}
-                  />
-                  <FormFeedback invalid>{emailError}</FormFeedback>
-                </InputGroup>
-              </FormGroup>
-              <FormGroup>
-                <InputGroup className="input-group-alternative">
-                  <InputGroupAddon addonType="prepend">
-                    <InputGroupText>
-                      <i className="ni ni-lock-circle-open" />
-                    </InputGroupText>
-                  </InputGroupAddon>
-                  <Input required valid
-                    type="password"                
-                    placeholder="Password"
-                    onChange ={(e) => setPassword(e.target.value)}
-                    autoComplete="off"  
-                  />
-                </InputGroup>
-              </FormGroup>
-              <FormGroup>
-                <InputGroup className="input-group-alternative-2" id="ConfirmPassword">
-                  <InputGroupAddon addonType="prepend">
-                    <InputGroupText>
-                      <i className="ni ni-check-bold" />
+                      <i className="ni ni-circle-08" />
                     </InputGroupText>
                   </InputGroupAddon>
                   <Input required valid
-                    class="is-invalid from-control"
-                    onChange={(e) => confirmpassword(e.target)}
-                    placeholder="Confirm Your Password"
-                    type="password"
+                    placeholder="IITK Roll no."
+                    type="roll"
                     autoComplete="off"
+                    onChange ={(e) => setRoll(e.target.value)}
```

Comment:
Make a popup if that roll number has been used. Something like an error: roll number is in use try to log in.